internal/db/update/update.go

```diff
@@ -58,10 +64,23 @@ func (m *SchemaUpdateManager) Schema() *SchemaUpdate {
 	return schema
 }
 
-func (m *SchemaUpdateManager) AppendSchema(extensions []schema.Update) {
-	m.updates[updateExternal] = extensions
+func (m *SchemaUpdateManager) AppendSchema(schemaExtensions []schema.Update, apiExtensions extensions.Extensions) {
```

Comment:
Let's try and make the name of the receiver var consistent. Right now it's sometimes called `m` and sometimes `s`. What do you think about choosing `s`? :)